tests/system/robot/chromeTests.py

```diff
@@ -55,52 +55,306 @@ def checkbox_labelled_by_inner_element():
 	)
 
 
+REVIEW_CURSOR_FOLLOW_CARET_KEY = ["reviewCursor", "followCaret"]
+
+
 def test_mark_aria_details():
+	spy = _NvdaLib.getSpyLib()
+	spy.set_configValue(REVIEW_CURSOR_FOLLOW_CARET_KEY, True)
+	exercise_mark_aria_details()
+
+
+def test_mark_aria_details_FreeReviewCursor():

```

Comment:
Thanks for introducing this additional test, however in most cases people who set review cursor not to follow the caret also disable navigator object following focus. Would it be possible to introduce a test making sure that in this case the behavior stays as expected?

Reply:
> also disable navigator object following focus

Which setting is that exactly? There is an option for "Review cursor follows focus", but I haven't found any settings about the navigator object following focus.

Reply:
I meant 'Review cursor follows focus'. Sorry for the confusion I didn't check how this option is named in the GUI.

Reply:
Thanks for clarifying, I thought it might be that, but wanted be sure.